src/LevelDBStore/IO/Data/LevelDB/DB.cs

```diff
@@ -35,37 +35,40 @@ public void Delete(WriteOptions options, byte[] key)
         public byte[] Get(ReadOptions options, byte[] key)
         {
             IntPtr value = Native.leveldb_get(handle, options.handle, key, (UIntPtr)key.Length, out UIntPtr length, out IntPtr error);
-            try
+
```

Comment:
Why remove try block?

Reply:
If something was wrong here we should stop the node, otherwise we return an empty array.

Reply:
But it used the `finally` block only. There was no `catch` block.

Reply:
I will revert it.